src/briefcase/commands/create.py

```diff
@@ -519,12 +519,10 @@ def install_app_code(self, app: BaseConfig):
 
         :param app: The config object for the app
         """
-
         # Remove existing app folder
         app_path = self.app_path(app)
-        if app_path.is_dir():
-            self.tools.shutil.rmtree(app_path)
-            self.tools.os.mkdir(app_path)
+        self.tools.shutil.rmtree(app_path)
```

Comment:
The `if is_dir` wasn't needed, as the app_path is *always* a directory, generated by the template.